msgpack-core/src/test/scala/org/msgpack/core/MessageUnpackerTest.scala

```diff
@@ -451,7 +470,7 @@ class MessageUnpackerTest extends MessagePackSpec {
         }
 
         block("v7") {
-          val unpacker = MessagePack.newDefaultUnpacker(data)
+          val unpacker = Random.shuffle(unpackers(data)).head
```

Comment:
I don't understand why do we need to create multiple MessageUnpacker instances and pick just one here. It adds the object instantiation cost and shuffling to the benchmark result. If we need to check the performance of individual MessageBuffer types, extract the code as

```
trait Fixture {
   val unpacker: MessageUnapacker
   var count = 0
   try {
       while (unpacker.hasNext) {
       readValue(unpacker)
       count += 1
   }
   finally {
       unpacker.close()
   }  
}

```

then reuse this trait:

```
block("v7-array") new Fixture {
   val unpacker = MessgePack.newDefaultUnpacker(data)
}

block("v7-bytebuffer") new Fixture {
    val unpacker = MessagePack.newUnpacker(bytebuffer)
}
```